packages/IBS/index.ts

```diff
@@ -33,6 +35,7 @@ async function initialise() {
 	keyring = new Keyring({ type: "sr25519" });
 	console.log(`Connected to CENNZnet network ${provider}`);
 	eve = keyring.addFromUri("//Eve");
+	ferdie = keyring.addFromUri("//Ferdie");
 }
```

Comment:
for `mainnet`, is it also as easy as `keyring.addFromUri("//Ferdie")`? Or it's gonna be different?

Reply:
For mainnet we'll need to store two private keys in the .env and create a keyring from those